packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -88,6 +88,7 @@ describe('TransactionSummary', () => {
   it('should ensure waitForResult always waits for the transaction to be processed', async () => {
     const { cleanup, ip, port } = await launchNode({
       args: ['--poa-interval-period', '750ms'],
+      poaInstant: false,
```

Comment:
Why was this necessary?

It seems unrelated to what's been done.

Reply:
@arboleya I think this is because `poaInstant` is `true` by default now, and it wasn't so earlier. `poaInstant` has to be `false` for us to use `--poa-interval-period`. Anyways, this should be solved when I remove all the extra params in favour of just `args`.

Reply:
When I say that `poaInstant` was `false` earlier, I mean the internal 'inferred' value of the flag. I think it just wasn't present in the list of the default args that we are passing in to the spawn command, but it is there now.